scripts/jenkinsProductionFiles.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# jenkinsProductionFiles.sh by Andrzej Pogonowski
+
+# Steps
+# 1. Removes all of the build and dependency directories (build, pack, node_modules)
+# 2. Copies all of the source files
+# 3. Copes configuration files and other necessary files
+
+# Cleanup
+rm -r pack
+rm -r build
+rm -r node_modules
+mkdir pack
+chmod -R 755 pack
+
+# Copy needed directories
+cp -rf ./src ./pack/src
+cp -rf ./cypress ./pack/cypress
+cp -rf ./.storybook ./pack/.storybook
+cp -rf ./envConfig ./pack/envConfig
+cp -rf ./public ./pack/public
+
+# Copy the needed files in the root directory
```

Comment:
Could you list why these files are "needed" in the PR somewhere, at a glance lots of this isn't explicitly needed for "production"

Reply:
I'm bringing a lot for safety. I'm not 100% sure if any of these files are actually needed. I just don't want to break the test environment when this is finally going to be connected end to end. I will be doing cleanup and refactoring issues right after I get this thing to work.

Reply:
Makes sense